tools/eventgen/main.go

```diff
@@ -0,0 +1,174 @@
+package main
+
+import(
+	"fmt"
+	"io/ioutil"
+	"os"
+	"time"
+
+	"github.com/BurntSushi/toml"
+)
+
+// Function for generating the front matter of an event file
```

Comment:
This isn't quite accurate, as it also includes the first part of the page too, with a standard header with the event's details. I'd suggest updating the comment.

Reply:
done